modules/ROOT/pages/endpoint-sftp-receive-target.adoc

```diff
@@ -1,91 +1,72 @@
-|Yes 
-
-|Size check wait time
-| The wait time in milliseconds between size checks to determine if a file is ready to be processed. This allows a file write to complete before processing.
+| Owner
+| The trading partner or host that owns the endpoint.
+| Yes
 
-When enabled, Mule performs two size checks by waiting the amount of time specified in between calls. To disable this feature, specify a negative number or omit the value. The default value is `1 second`.
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.
+| Yes
 
-If both checks return the same value, the file is ready to process. 
-|Yes 
+| Description
+| The user-defined field for describing the purpose and use of the endpoint.
+| No
 
-|Archival Path
-| Specify an optional path on the SFTP server for where to store a copy of the file that was picked up from the `source path`. 
-|
-|===
+| Host
+| The SFTP server host, such as `sftp.mulesoft.com`, `localhost`, or `192.168.0.1`. The host must be accessible from your Mule environment.
+| Yes
 
-== SFTP Acknowledgment Endpoint Settings
+| Port
+| The port number of the SFTP server.
+| Yes
 
-The configuration options for the acknowledgment endpoints are the same as the SFTP Receive endpoint, except that the `Source Path` and the `Archival Path` are replaced by the `Acknowledgement Path`.
+| Username
+| The username for the SFTP Server.
+| Yes, if the server is authenticated.
 
-Use this setting to specify the SFTP endpoint to which to send acknowledgement files (TA1 and/or 997/999) to your trading partners:
+| Password
+| The password for the SFTP Server.
+| Yes, if the server is authenticated.
 
-[%header%autowidth.spread]
-|===
-|Name |Description |Required
+| Source path
+| The path on the SFTP server to which your trading partner sends  the inbound files.
+| Yes
 
-|Acknowledgement Path
-| A path on the SFTP server where your trading partner is expecting to receive an acknowledgement (such as TA1 and/or 997/999) related to an EDI transaction they may have sent.
-|Yes
-|===
+| Polling Frequency
+| The frequency in which the server checks the source path for new files. The value in this field is set to `1 minute` by default.
+| Yes
 
-== SFTP Target Endpoint Settings
+| Size check wait time
+| The wait time in seconds between size checks to determine if a file is ready for processing.. The value in this field is set to `1 second` by default. This allows a file write to complete before processing.
 
-The configuration options for the acknowledgment endpoints are the same as the SFTP Receive endpoint, except that the `Source Path` and the `Archival Path` are replaced by the `Target Path`.
+When enabled, Mule performs two size checks by waiting the amount of time specified in the `Size check wait time` field. If both checks return the same value, the file is ready to process.
```

Comment:
Misplaced modifier: Mule is not the thing enabled.
Specify what is enabled.

Reply:
Removed - I think this is a superfluous clause.